src/preprocessor.ts

```diff
@@ -48,15 +46,12 @@ export function process(
             }
         );
 
-        const outputText =
-            compilerOptions.allowSyntheticDefaultImports && !tsJestConfig.skipBabel
-                ? babelJest.process(
+        const outputText = postHook(
                     tsTranspiled.outputText,
                     path + '.js', // babel-jest only likes .js files ¯\_(ツ)_/¯
```

Comment:
Damnit, I realised this I had forgotten this just as I left.
Actually to fix #211 we should just not modify the path - we don't need to fix it to pass babel's typechecks anymore.